go/tasks/plugins/k8s/kfoperators/common/config.go

```diff
@@ -0,0 +1,32 @@
+package common
+
+import (
+	"time"
+
+	pluginsConfig "github.com/flyteorg/flyteplugins/go/tasks/config"
+	"github.com/flyteorg/flytestdlib/config"
+)
+
+//go:generate pflags Config --default-var=defaultConfig
+
+var (
+	defaultConfig = Config{
+		Timeout: config.Duration{Duration: 1 * time.Minute},
+	}
+
+	configSection = pluginsConfig.MustRegisterSubSection("pytorch", &defaultConfig)
```

Comment:
if this is in `common` it shouldn't be named `pytorch` right?

Reply:
sorry, my bad, updated it